docs/faq/coordinators.md

```diff
@@ -1,39 +1,69 @@
+## Auction Process
+
+### How do I become a Coordinator?
+
+To become a coordinator you need to prepare the system infrastructure, take part in the auction and win a bid for a time slot.
+
+### How bids can be placed in the auction?
+
+Coordinators will participate in the auction by sending an on-chain transaction to the auction smart contract. Bids are always payed in HEZ
 
-To become a coordinator you need to prepare the system infrastructure and take part in an auction and win a bid for a time slot.
+### Where do I get HEZ?
 
-## How long are coordinator time slots?
+HEZ can be easily obtained in [Uniswap](https://app.uniswap.org/)
 
-Slots will be 40 Ethereum blocks long (10 minutes).
+### How long are Coordinator time slots?
 
-## What happens if the coordinator goes offline?
+Slots will be 40 Ethereum blocks long (10 minutes). During this time, a Coordinator can forge as many batches as possible.
 
-If the coordinator has not done anything in the first part a slot (TBD), then anyone can jump in and replace it by forging blocks (first come first served).
+### What happens if the Coordinator goes offline?
 
-The information stored on-chain (as part of [calldata](https://ethereum.stackexchange.com/a/52992)) is enough to allow anyone to independently build the full state tree (and become a coordinator themselves).
+If the Coordinator has not done anything in the first part a slot, then anyone can jump in and replace it by forging blocks (first come first served).
 
-## How do coordinators make money?
+### How do Coordinators make money?
 
-They collect the transaction fees. They can expect a revenue per transaction and each coordinator will select to forge the more profitable transactions from the transaction pool.
+Coordinators set and collect the transaction fees. They can expect a revenue per transaction and each coordinator will select to forge the more profitable transactions from the transaction pool.
 
 They profit from these fees minus the operational costs and the bid price for the slot.
 
-## How bids can be placed in the auction?
+## Running a Coordinator
+
+### What are the infrastructure requirements to run a Coordinator?
+
+To run a Coordinator you need the following:
+- PostgreSQL database
+- Ethereum Node
+- Coordinator Server running the [hermez-node repository](https://github.com/hermeznetwork/hermez-node). A reasonable server spec is AWS instance c5ad.2xlarge with 8 vCPU and 16GB RAM
+- Proof Server running the [rapidsnark repository](https://github.com/iden3/rapidsnark). A server spec able to run the ~2000 transaction circuit is AWS instance c5ad.24xlarge with 96 vCPU and 192GB RAM. 
+
+### Is there a step by step guide to configure a Coordinator node?
+
+Yes. You can find this guide [here](../developers/coordinator.md).
+
+### How many transactions can be processed per batch?
+ 
+Number of transactions per batch depend on the circuit used. Hermez accepts two different circuit sizes: 344 transactions and 1912 transactions. 
```

Comment:
Additional questions to consider adding:

1.) I'm having technical issues ("as a Coordinator" or "running a coordinator node")  who can I contact for support?
2.)Is there requirement for the number of HEZ tokens you have to hold to be a Coordinator? 
3.) Will there be a public registration of the Hermez Network Coordinators?